customcommands/bot.go

```diff
@@ -117,6 +117,91 @@ type DelayedRunCCData struct {
 	IsExecedByLeaveMessage bool `json:"is_execed_by_leave_message"`
 }
 
+var cmdEvalCommand = &commands.YAGCommand{
+	CmdCategory:  commands.CategoryTool,
+	Name:         "Evalcc",
+	Description:  "executes small custom command code",
+	RequiredArgs: 1,
+	Arguments: []*dcmd.ArgDef{
+		{Name: "code", Type: dcmd.String},
+	},
+	SlashCommandEnabled: false,
+	DefaultEnabled:      true,
+	RunFunc: func(data *dcmd.Data) (interface{}, error) {
+		var hasCoreWriteRole bool
```

Comment:
```suggestion
		hasCoreWriteRole := false
```
IMO this is a little better as it makes the initial value (`false`) more obvious, but up to you.